index.html

```diff
@@ -125,6 +127,33 @@
     </section>
 
     <section id='sotd'>
+      <p>
+The Working Group thanks the following individuals for their contributions 
+toward not only the content of this document, but yeoman's work in this
+standards community that drove changes, discussion, and consensus among a sea 
+of varied opinions: Matt Stone, Gregg Kellogg, Ted Thibodeau Jr, Oliver Terbu, 
+Joe Andrieu, David I. Lehn, Matthew Collier, and Adrian Gropper.
+      </p>
+
+      <p>
+Work on this specification has been supported by the Rebooting the
+Web of Trust community facilitated by Christopher Allen, Shannon Appelcline,
+Kiara Robles, Brian Weller, Betty Dhamers, Kaliya Young, Manu Sporny,
+Drummond Reed, Joe Andrieu, Heather Vescent, Samantha Chase, and Andrew Hughes.
+The participants in the Internet Identity Workshop facilitated by 
+Phil Windley, Kaliya Young, Doc Searls, and Heidi Nobantu Saul also 
+supported the refinement of this work through numerous working sessions 
+designed to educate, debate, and improve the content in this specification.
```

Comment:
`designed to educate, debate, and improve the content in this specification` needs some rework, as it currently reads to me as a shortening of `designed to educate the content, debate the content, and improve the content in this specification`.

Maybe `designed to educate about, debate on, and improve the content of this specification`, as these verb forms can all reasonably apply to either "this specification" or "the content of"?

Reply:
Fixed in ab9a5322046d152943b2ff5e4f51002e1fc76760.